src/NServiceBus.Core/Recoverability/Faults/ErrorQueueSettings.cs

```diff
@@ -11,6 +11,12 @@ namespace NServiceBus
     public static class ErrorQueueSettings
     {
         /// <summary>
+        /// The settings key where the error queue address is stored.    
+        /// </summary>
+        public const string SettingsKey = "errorQueue";
+        const string DefaultErrorQueueName = "error";
```

Comment:
discussed this with @andreasohlund on a call and given his argument about the readability of the code with the current constraints (checking alternative configuration sources for error queue configuration) I'm good with that approach.